### PR TITLE
[TRUS-3106] - Residency,Nationality for Deceased Settlor in Trust Registration

### PR DESCRIPTION
**Pull request by jassalrichy**
Rename packages
Check answers
CountryOfResidenceYesNo, CountryOfResidenceInTheUkYesNo & CountryOfResidence
Update Navigation
Mapper

### CODE REVIEW

File: app/controllers/deceased_settlor/mld5/CountryOfResidenceController.scala [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                                                   view: CountryOfResidenceView,
+                                                   val countryOptions: CountryOptionsNonUK
+                                 )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+
+  val form: Form[String] = formProvider.withPrefix("5mld.countryOfResidence")
+
+  def onPageLoad(mode: Mode, draftId: String): Action[AnyContent] = (actions.authWithData(draftId) andThen requireName(draftId)) {
+    implicit request =>
+
+      val name = request.userAnswers.get(SettlorsNamePage).get
```
Comment (jackheslop96) [inline app/controllers/deceased_settlor/mld5/CountryOfResidenceController.scala:55]: Having done requireName can you not retrieve the name from the request?

File: app/utils/CheckYourAnswersHelper.scala [hunk 2]
```diff
@@ -328,4 +340,118 @@ class CheckYourAnswersHelper @Inject()(countryOptions: CountryOptions,
...
+//    x =>
+//      AnswerRow(
+//        "settlorDateOfBirthYesNo.checkYourAnswersLabel",
+//        yesOrNo(x),
+//        Some(controllers.deceased_settlor.routes.SettlorDateOfBirthYesNoController.onPageLoad(NormalMode, draftId).url),
+//        deceasedSettlorName(userAnswers),
+//        canEdit = canEdit
+//      )
+//  }
+
```
Comment (jackheslop96) [inline app/utils/CheckYourAnswersHelper.scala:456]: Can this be deleted?

File: app/controllers/deceased_settlor/mld5/CountryOfResidenceController.scala (hunk 1 above)
Comment (jassalrichy) [reply app/controllers/deceased_settlor/mld5/CountryOfResidenceController.scala]: Updated controllers

File: app/utils/CheckYourAnswersHelper.scala (hunk 2 above)
Comment (jassalrichy) [reply app/utils/CheckYourAnswersHelper.scala]: Removed

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**